x/operator/types/keys.go

```diff
@@ -155,6 +209,31 @@ func KeyForOperatorKeyRemovalForChainID(addr sdk.AccAddress, chainID string) []b
 	)
 }
 
+func ParseKeyForOperatorKeyRemoval(key []byte) (addr sdk.AccAddress, chainID string, err error) {
+	// Check if the key has at least 20 byte for the operator and eight byte for the chainID length
+	if len(key) < AccAddressLength+8 {
```

Comment:
The  `AccAddressLength + 8` is recommended to be extracted as a variable or constant to enhance maintainability.

Reply:
I will use a constant to replace 8 in the next commit.